index.html

```diff
@@ -32,22 +32,30 @@
     
     <body ng-controller="bodyController">
         <div class="header-wrapper" ng-controller="HeaderController">
-            <nav class="navbar navbar-default navbar-fixed-top navbar-custom">
-                <div class="container-fluid">
-                    <div class="navbar-header">
-                        <a class="header-text navbar-brand" id="product-name" ng-click="goToRoute('/income')"><img src="/img/bowl.png" class="brand-logo"></a>
-                        <a class="header-text navbar-brand" id="product-name" ng-click="goToRoute('/income')"><span>Plenti</span></a>
+            <div class="navbar-wrapper">
+                <nav class="navbar navbar-default navbar-fixed-top navbar-custom">
+                    <div class="container-fluid">
+                        <div class="navbar-header">
+                            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse-1">
+                                <span class="sr-only">Toggle navigation</span>
+                                <span class="icon-bar"></span>
+                                <span class="icon-bar"></span>
+                                <span class="icon-bar"></span>
+                              </button>
+                            <a class="header-text navbar-brand" ng-click="goToRoute('/income')">

```

Comment:
I'm sure there's a reason, but why make this an `ng-click` instead of just `href`, considering it's an `<a>` tag?